dkg/dkg.go

```diff
@@ -38,6 +38,11 @@ import (
 	"github.com/obolnetwork/charon/tbls/tblsconv"
 )
 
+const (
+	DutyLock        core.DutyType = 101
+	DutyDepositData core.DutyType = 202
```

Comment:
do not export this

Reply:
this should probably be `101 and 102`

Also add a comment that this cannot change since it breaks backwards compatibility.